admin/c2cgeoportal_admin/views/layer_groups.py

```diff
@@ -5,13 +5,15 @@
 from c2cgeoform.views.abstract_views import ListField
 from c2cgeoportal_commons.models.main import LayerGroup
 from c2cgeoportal_admin.schemas.treegroup import children_schema_node
+from c2cgeoportal_admin.schemas.metadata import metadatas_schema_node
 from c2cgeoportal_admin.views.treeitems import TreeItemViews
 
 _list_field = partial(ListField, LayerGroup)
 
 
 base_schema = GeoFormSchemaNode(LayerGroup)
 base_schema.add(children_schema_node())
+base_schema.add(metadatas_schema_node.clone())
```

Comment:
maybe no need for a clone

Reply:
I prefer to keep clone() in case we alter it later, it will not alter schema in other views.